### PR TITLE
docs(Gamut): update headings to sentence case

### PR DESCRIPTION
**Pull request by uxemilylee**
### Overview

Update all headings in Gamut Storybook to sentence case for consistency with our Codecademy writing guidelines

### PR Checklist

- [ ] Related to designs: n/a
- [x] Related to JIRA ticket: [GM-796]
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

[GM-796]: https://skillsoftdev.atlassian.net/browse/GM-796?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6690858bd9aef2881724c9e9--gamut-preview.netlify.app>https://6690858bd9aef2881724c9e9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6690858bd9aef2881724c9e9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@66.29.3-alpha.832860.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://669597f7a7f8ebc9b35bae60--gamut-preview.netlify.app>https://669597f7a7f8ebc9b35bae60--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/669597f7a7f8ebc9b35bae60>Deploy Logs</a></p>

File: packages/styleguide/stories/Organisms/GridForm.stories.mdx [hunk 1]
```diff
@@ -485,7 +485,7 @@ the <LinkTo id="atoms-button--fill-button">`FillButton`</LinkTo> or <LinkTo id="
   </Story>
 </Canvas>
 
-## Inline Submit Button
+## Inline submit button
 
 We can make the Submit button inline with an input by setting the column
```
Comment (LinKCoding) [inline packages/styleguide/stories/Organisms/GridForm.stories.mdx:490]: should "Submit" here be lower cased?

File: packages/styleguide/stories/UX Writing/About.stories.mdx [hunk 2]
```diff
@@ -21,7 +21,7 @@ In the following pages, you’ll find a collection of tips and tools for writing
 
 ## Resources
 
-### External Tools & Resources:
+### External tools and resources:
 
 🖍 **[Hemingway App](https://hemingwayapp.com/):** Tool for checking copy for grade level and readability
```
Comment (LinKCoding) [inline packages/styleguide/stories/UX Writing/About.stories.mdx:27]: (not sure why I can't commit directly below) but for the capitalize my title resource — should we omit that now? 

Similarly is there any references/notes you want to make about heading titles and story names?

**Review by LinKCoding — COMMENTED** [review]
This looks great! 
Holding off on approval until we fix merge conflicts tho! 

You can also feel free to change this PR from draft to ready for review!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@67.0.6-alpha.e9d97f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://66abe1160175d34c2e536673--gamut-preview.netlify.app>https://66abe1160175d34c2e536673--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/66abe1160175d34c2e536673>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@67.1.5-alpha.d9482c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://66ba7b3310d9f53b9e605855--gamut-preview.netlify.app>https://66ba7b3310d9f53b9e605855--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/66ba7b3310d9f53b9e605855>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@67.1.5-alpha.89f131.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://66bbb6cfab6023a03e659a28--gamut-preview.netlify.app>https://66bbb6cfab6023a03e659a28--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/66bbb6cfab6023a03e659a28>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]

**Pull request merged**